lib/python/vdm/server/HTTPListener.py

```diff
@@ -438,8 +480,681 @@ def map_deployment_users(request, user):
+        DEPLOYMENT[i]['users'] = {}
+        DEPLOYMENT[i]['users']['user'] = []
+        deployment_user = filter(lambda t: t['databaseid'] == DEPLOYMENT[i]['databaseid'], DEPLOYMENT_USERS)
+        if len(deployment_user) == 0:
+            DEPLOYMENT[i]['users'] = None
+        for user in deployment_user:
+            DEPLOYMENT[i]['users']['user'].append({
+                'name': user['name'],
+                'roles': user['roles'],
+                'plaintext': user['plaintext'],
+                'password': user['password'],
+                'databaseid': user['databaseid']
+            })
+
+        deployment_elem = SubElement(deployment_top, 'deployment')
+        for key, value in DEPLOYMENT[i].iteritems():
+            if type(value) is dict:
+                handle_deployment_dict(deployment_elem, key, value, False)
+            elif type(value) is list:
+                handle_deployment_list(deployment_elem, key, value)
+            else:
+                if value is not None:
+                    deployment_elem.attrib[key] = str(value)
+        i += 1
+
+    try:
+        # f = open(PATH + 'vdm.xml' if PATH.endswith('/') else PATH + '/' + 'vdm.xml','w')
+        # vdm_path = 'vdm.xml' if PATH.endswith('/') else PATH + '/' + 'vdm.xml'
+        path = os.path.join(PATH, 'vdm.xml')
+        f = open(path, 'w')
+        f.write(tostring(main_header,encoding='UTF-8'))
+        f.close()
+
+
+
+    except Exception, err:
+        print str(err)
+
+
+def sync_configuration():
+    headers = {'content-type': 'application/json'}
+    url = 'http://'+__IP__+':'+str(__PORT__)+'/api/1.0/vdm/configuration/'
+    response = requests.post(url,headers = headers)
+    return response
+
+def convert_xml_to_json(config_path):
+    with open(config_path) as f:
+        xml = f.read()
+    o = XML(xml)
+    xml_final = json.loads(json.dumps(etree_to_dict(o)))
+    if type(xml_final['vdm']['members']['member']) is dict:
+        member_json = get_member_from_xml(xml_final['vdm']['members']['member'], 'dict')
+    else:
+        member_json = get_member_from_xml(xml_final['vdm']['members']['member'], 'list')
+
+    if type(xml_final['vdm']['databases']['database']) is dict:
+        db_json = get_db_from_xml(xml_final['vdm']['databases']['database'], 'dict')
+    else:
+        db_json = get_db_from_xml(xml_final['vdm']['databases']['database'], 'list')
+
+    if type(xml_final['vdm']['deployments']['deployment']) is dict:
+        deployment_json = get_deployment_from_xml(xml_final['vdm']['deployments']['deployment'], 'dict')
+    else:
+        deployment_json = get_deployment_from_xml(xml_final['vdm']['deployments']['deployment'], 'list')
+    if type(xml_final['vdm']['deployments']['deployment']) is dict:
+        user_json = get_users_from_xml(xml_final['vdm']['deployments']['deployment'], 'dict')
+    else:
+        user_json = get_users_from_xml(xml_final['vdm']['deployments']['deployment'], 'list')
+
+    global DATABASES
```

Comment:
For the global variables, it might be cleaner to use a class similar to how it is done in lib/python/voltcli/utility.py